tests/cpu/compare_with_acts_seeding.cpp

```diff
@@ -70,10 +70,10 @@ inline bool operator==(const Acts::BoundVector& acts_vec,
             traccc::float_epsilon * 10 &&
         std::abs(acts_vec[Acts::eBoundTheta] -
                  traccc::getter::element(traccc_vec, traccc::e_bound_theta,
-                                         0)) < traccc::float_epsilon * 10 &&
+                                         0)) < traccc::float_epsilon * 1000 &&
```

Comment:
I had to mitigate the condition for equality operation on momentum direction as the specific bug is not fixed in ACTS yet.
I will revert the change once acts-project/acts#2211 is included in any of next releases

Reply:
This can be reverted now with https://github.com/acts-project/acts/releases/tag/v27.0.0